<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlagsDialog</class>
 <widget class="QDialog" name="FlagsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Features</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="mCBTime">
            <property name="text">
             <string>Time</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mCBGyroscope">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Gyroscope</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mCBAccelerometer">
            <property name="text">
             <string>Accelerometer</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mCBQuaternion">
            <property name="text">
             <string>Quaternion</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mCBPitch">
            <property name="text">
             <string>Pitch</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mCBYaw">
            <property name="text">
             <string>Yaw</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mCBRoll">
            <property name="text">
             <string>Roll</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mCBEmg">
            <property name="text">
             <string>Emg</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="mCBNormalize">
            <property name="text">
             <string>Normalize</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mCBAllPositive">
            <property name="text">
             <string>All Positive</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Sempling</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="mRBSempleMode">
          <property name="text">
           <string>Sample per row</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="mGBSemplePerRow">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="mLBReps">
             <property name="text">
              <string>N semple per row</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="mSBReps">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mRBGestureMode">
          <property name="text">
           <string>Gesture per row</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="mGBGesturePerRow">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Time per gesture</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="mDSBTimePerGesture">
             <property name="minimum">
              <double>0.250000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.250000000000000</double>
             </property>
             <property name="value">
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Delta time of a sample</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="mDSBDeltaTimeOfASemple">
             <property name="minimum">
              <double>0.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.050000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="mPBCancel">
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="mPBOk">
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="mVSBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mPBOk</sender>
   <signal>clicked()</signal>
   <receiver>FlagsDialog</receiver>
   <slot>onOk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPBCancel</sender>
   <signal>clicked()</signal>
   <receiver>FlagsDialog</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOk()</slot>
  <slot>onCancel()</slot>
 </slots>
</ui>
